website/data/docs-remote-plugins.json

```
@@ -1,4 +1,10 @@
 [
+  {
+    "title": "Alicloud",
+    "path": "alicloud",
+    "repo": "hashicorp/packer-plugin-alicloud",
+    "version": "latest"
+  },
```

With the latest master we can now mark plugins as community

```suggestion
    "pluginTier": "community",
    "version": "latest"
  },
```